src/baguetteBox.js

```diff
@@ -148,50 +146,75 @@
         supports.svg = testSVGSupport();
 
         buildOverlay();
+        destroyGalleriesForSelector(selector);
```

Comment:
When we call run() function multiple times for the same selector we want to destroy any cached data for selector and unbind its listeners.